.stylelintrc.json

```diff
@@ -20,7 +20,7 @@
       "/^box-shadow": ["px"],
       "/^margin/": ["rem"],
       "/^padding/": ["rem"],
-      "font-size": ["px", "em"],
+      "font-size": ["px", "rem", "em"],
```

Comment:
I think we should disallow `px` generally and only override it for the browser extension.

Also we should probably add `line-height` here too to make sure it is scaled together with the font-size (and doesn't use px, but only unit-less or `em`).